src/spicy/runtime-support.cc

```diff
@@ -774,10 +776,10 @@ std::string rt::fuid() {
     throw ValueUnavailable("fuid() not available in current context");
 }
 
-std::string rt::file_begin(const std::optional<std::string>& mime_type) {
+std::string rt::file_begin(const std::optional<std::string>& mime_type, const std::optional<std::string>& fuid) {
```

Comment:
Since down the line we take ownership anyway let's take a value here and `std::move` on `push` below.

Reply:
As just discussed in chat - this is sadly not possible, since `spicyz` expects that this is passed by reference.